packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx

```diff
@@ -10,10 +10,12 @@ const StyledDropdownMenu = styled.div<{
       ? 'none'
       : 'blur(12px) saturate(200%) contrast(50%) brightness(130%)'};
 
+  color: ${({ theme }) => theme.font.color.secondary};
+
   background: ${({ theme, disableBlur }) =>
     disableBlur
       ? theme.background.primary
-      : theme.background.transparent.secondary};
+      : theme.background.transparent.primary};
```

Comment:
the background looks correct according to the ticket (transparent.secondary)

Reply:
Why are we changing this?

Reply:
> the background looks correct according to the ticket (transparent.secondary)

the current background is more lighter which makes it a bit different in looks when compared to that of menu bars opened when click on table action buttons.

since, the requirement was to make all menu bar follow same theme. that's why i did change the background color as well. (although the bg used in table menus on the left uses theme.background.primary | i went for transparent.primary as we are also having some blur css so wanted to keep its point).

else, i can revert back to original background and show you the difference if you want.